raiden/tests/utils/blockchain.py

```diff
@@ -328,16 +328,24 @@ def geth_create_blockchain(
 
         processes_list.append(process)
 
-    geth_wait_and_check(deploy_client, private_keys, random_marker)
+    try:
+        geth_wait_and_check(deploy_client, private_keys, random_marker)
+
+        # reenter echo mode (disabled by geth pasphrase prompt)
+        if isinstance(sys.stdin, io.IOBase):
```

Comment:
the function above `geth_wait_and_check` may throw, and this won't be executed. Could you please move it to a finally block?